src/app/fdctl/run/topos/fd_frankendancer.c

```diff
@@ -361,6 +361,11 @@ fd_topo_initialize( config_t * config ) {
       tile->net.quic_transaction_listen_port   = config->tiles.quic.quic_transaction_listen_port;
       tile->net.legacy_transaction_listen_port = config->tiles.quic.regular_transaction_listen_port;
 
+      /* multihome support */
+      ulong multi_cnt = tile->net.multihome.cnt = config->tiles.net.multihome.src_ip_addr_cnt;
+      for( ulong j = 0; j < multi_cnt; ++j ) {
+        tile->net.multihome.src_ip_addr[j] = config->tiles.net.multihome.src_ip4_addr[j];
```

Comment:
does this need to be a `fd_memcpy`?

Reply:
fd_memcpy is not great for small copies. Also, we expect multi_cnt to be 0 or 1.
Lastly, this is in initialization where this kind of concern is unimportant.